react/AppTile/index.jsx

```diff
@@ -52,14 +55,30 @@ export const AppTile = ({
     ? showStatus.indexOf(statusLabel) > -1 && statusLabel
     : showStatus && statusLabel
   IconComponent = IconComponent || AppIcon
+  const isInMaintenance = maintenanceDisplay && statusLabel === 'maintenance'
```

Comment:
It is not enough. Take a look at `isUnderMaintenance`

Reply:
Hmm above there is getCurrentStatusLabel (statusLabel)  which calls `isUnderMaintenance` (which returns 'maintenance'). 

So it's ok.